driver/CBAInferFusion_driver.hpp

```diff
@@ -601,7 +601,7 @@ int CBAInferFusionDriver<Tgpu, Tref>::AllocateBuffersAndCopy()
     else
         out_sz = in_sz; // This is for N+A so the output is the same as the input size
 
-    if(miopen::IsEnabled(ENV(MIOPEN_DRIVER_PAD_BUFFERS_2M)))
+    if(env::enabled(MIOPEN_DRIVER_PAD_BUFFERS_2M))
```

Comment:
[R][Quality] Removal of "Is" works with "enabled" and "disabled", but this is not good as a general solution. For example "unset(VAR)" is ambiguous: is it not clear if it queries the state of VAR or changes it.

Reply:
A pitfall of irregular verb "set". I agree, that may be confusing.

Reply:
There is no `env::unset()` function. To clear a variable (removes from a process environment variables), one needs to call `env::clear()`. To check if the variable exists (has been set in the environment), one needs to query the name, e.g., `if (MIOPEN_ALL_TESTS) { ... }` for checking if `MIOPEN_ALL_TESTS` is set, and `if(!MIOPEN_ALL_TESTS) { ... }` for checking is it is not set.